src/docs/examples/notlogged_toplayer.html

```diff
@@ -6,42 +6,43 @@
 
 </head>
 <body>
-<div class="mint-toplayer mint-toplayer--modal mint-toplayer--medium mint-toplayer--lead mint-toplayer--with-bugbox">
+<div class="mint-overlay">
```

Comment:
@mr-mig thanks to this only one element has ```position:fixed```, one element has to be removed on close and we can use flexbox for positioning elements inside an overlay.